literature.bib

```diff
@@ -6967,6 +7268,204 @@ @article{DBW:SCP18
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2017 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@inproceedings{FSS:GPCE17,
+	author = {Fenske, Wolfram and Schulze, Sandro and Saake, Gunter},
+	title = {{How Preprocessor Annotations (Do Not) Affect Maintainability: A Case Study on Change-Proneness}},
+	booktitle = GPCE,
+	publisher = ACM,
+	address = NY,
+	pages = {77--90},
+	doi = {10.1145/3136040.3136059},
+	month = OCT,
+	year = 2017,
+	ek-tags = {C preprocessor}
+}
+
+@inproceedings{FZB:SPLC17,
+	author = {Fu{\ss}berger, Nicolas and Zhang, Bo and Becker, Martin},
+	title = {{A Deep Dive into Android's Variability Realizations}},
+	booktitle = SPLC,
+	publisher = ACM,
+	address = NY,
+	pages = {69--78},
+	doi = {10.1145/3106195.3106213},
+	year = 2017
+}
+
+@inproceedings{HGM+:ICSSP17,
+	author = {Hohl, Philipp and Ghofrani, Javad and M{\"{u}}nch, J{\"{u}}rgen and Stupperich, Michael and Schneider, Kurt},
+	title = {{Searching for Common Ground: Existing Literature on Automotive Agile Software Product Lines}},
+	booktitle = ICSSP,
+	publisher = ACM,
+	address = NY,
+	pages = {70--79},
+	doi = {10.1145/3084100.3084109},
+	year = 2017,
+	ek-tags = {industry}
+}
+
+@inproceedings{HPF:SPLC17,
+	author = {Horcas, Jose-Miguel and Pinto, M{\'{o}}nica and Fuentes, Lidia},
+	title = {{Green Configurations of Functional Quality Attributes}},
+	booktitle = SPLC,
+	publisher = ACM,
+	address = NY,
+	pages = {79--83},
+	doi = {10.1145/3106195.3106205},
+	year = 2017,
+	ek-tags = {sustainability}
+}
+
+@misc{ISO24765:17,
+	author = {ISO/IEC/IEEE},
+	title  = {{Systems and Software Engineering -- Vocabulary}},
+	type   = {International Standard},
+	number = {24765:2017},
+	doi    = {10.1109/IEEESTD.2017.8016712},
+	year   = 2017
+}
```

Comment:
Maybe put this entry down into the `misc` sections where all the tools are? I don't think we have a real policy here so I am fine either way.

Reply:
True. Fine with me.